tests/unit/config/test_configinit.py

```diff
@@ -65,6 +67,22 @@ def configdata_init(monkeypatch):
 
 class TestEarlyInit:
 
+    def test_config_py_path(self, args):
```

Comment:
You'll need to use the `init_patch` fixture here (like `test_config_py` does) so stuff which is in the way gets patched away.